src/ScriptVerification/SucceedIfNumber.agda

```diff
@@ -170,8 +170,8 @@ opaque
   succeedExample : ComputationResult String UTxOState
   succeedExample = UTXO-step initEnv ⟦ initStateDatum , 0 , ∅ , 0 ⟧  succeedTx
 
-  _ : isSuccess succeedExample ≡ true
-  _  = refl
+  -- _ : isSuccess succeedExample ≡ true
```

Comment:
Does this not succeed anymore or is this just a leftover? I feel like if this doesn't work anymore it indicates a problem.

Reply:
This was an oversight